java/client/src/main/java/glide/managers/ConnectionManager.java

```diff
@@ -154,15 +169,7 @@ private ConnectionRequestOuterClass.ReadFrom mapReadFromEnum(ReadFrom readFrom)
 
     /** Check a response received from Glide. */
     private Void checkGlideRsResponse(Response response) {
```

Comment:
This function really doesn't do anything anymore.

Reply:
+1